cmd/nerdctl/network_inspect_test.go

```diff
@@ -66,3 +67,31 @@ func TestNetworkInspect(t *testing.T) {
 	}
 	assert.DeepEqual(base.T, expectedIPAM, got.IPAM)
 }
+
+func TestNetworkWithNamespace(t *testing.T) {
+	testutil.DockerIncompatible(t)
+
+	t.Parallel()
+
+	tID := testutil.Identifier(t)
+	base := testutil.NewBase(t)
+	baseOther := testutil.NewBaseWithNamespace(t, tID)
+
+	tearDown := func() {
+		base.Cmd("network", "rm", tID).Run()
+		baseOther.Cmd("network", "prune", "-f").Run()
+	}
+	tearDown()
+	t.Cleanup(tearDown)
+
+	base.Cmd("network", "create", tID).AssertOK()
+
+	// Other namespace cannot inspect, prune, see, or remove this network
+	baseOther.Cmd("network", "inspect", tID).AssertFail()
+	baseOther.Cmd("network", "prune", "-f").AssertOutNotContains(tID)
+	baseOther.Cmd("network", "ls").AssertOutNotContains(tID)
+
+	if !rootlessutil.IsRootless() {
```

Comment:
Why?

Reply:
I actually do not know.
This was already in the original test: https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/network_remove_linux_test.go#L53

Is there any condition under which remove would not work for rootless?

Reply:
Looks like it was from here: https://github.com/containerd/nerdctl/commit/a4b51e2842d95800262fb162de43c2cab26fce1f

so - recent.

Maybe it was just a fluke? I removed it and it is ok locally.